platform/ui/src/components/Thumbnail/Thumbnail.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '@ohif/ui';
+
+const Thumbnail = ({
+  seriesDescription,
+  seriesNumber,
+  instanceNumber,
+  viewportIdentificator,
+  isTracked,
+  isActive,
+  onClick,
+}) => {
+  const trackedIcon = isTracked ? 'circled-checkmark' : 'dotted-circle';
+
+  return (
+    <div
+      className="flex flex-row flex-1 px-4 py-2 showExcludeButtonOnHover cursor-pointer"
+      onClick={onClick}
+    >
+      <div className="flex flex-col flex-2 px-2 items-center">
+        <div
+          className={classnames(
+            'flex flex-col items-center justify-start p-2 mb-2 relative cursor-pointer',
+            isTracked && 'rounded-sm hover:bg-gray-900 showTooltipOnHover'
+          )}
+        >
+          <Icon
+            name={trackedIcon}
+            className="text-primary-light mb-2"
+            style={{ width: '19px' }}
```

Comment:
Same as height (above), if possible and if this is not very specific, we can maybe replace this for `w-5`.